docs/03-03-examples-alpha.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Kyma CLI alpha command usage examples
+type: Details
+---
+
+The following examples show how to provision a cluster, and to deploy and update Kyma with alpha commands.
+
+## Provision a cluster
+To provision a kubernetes cluster, run:
+
+```bash
+kyma alpha provision k3s 
+```
+
+
+## Install Kyma
+
+There are several ways to install Kyma:
+
+- To install Kyma on a local cluster, you can simply use the `deploy` command.<br>
+Kyma provides a default domain under the URL `https://console.local.kyma.dev`.
+
+    ```bash
+    kyma alpha deploy 
+    ```
```

Comment:
Once again, I have no idea why the code is not formatted properly in the [website](https://deploy-preview-764--kyma-project-cli-preview.netlify.app/cli/overview/#details-kyma-cli-alpha-command-usage-examples), but you can also omit the programming language and see if that makes a difference, or even write single ` instead of triple ``` 😟 
```suggestion
    ```
    kyma alpha deploy 
    ```
```